pypit/arsciexp.py

```diff
@@ -16,6 +16,7 @@
 from pypit import armasters
 from pypit import armsgs
 from pypit import arproc
+from pypit import arqa
```

Comment:
OK, I've moved this import into the routine where it is used. Would you rather I kept the calls to arqa separate from `arsciexp`?